src/app/donation-start/donation-start.component.ts

```diff
@@ -688,7 +688,7 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
 
   captchaReturn(captchaResponse: string) {
     this.captchaCode = captchaResponse;
```

Comment:
Might be helpful to sanity check that `captchaResponse` is not falsey in case a dev makes a change in the future which breaks this functionality without realising.